testutils/metahelper/client/client_test.go

```diff
@@ -99,8 +98,8 @@ func TestSync(t *testing.T) {
 		if (err == nil && data.expErr) || (err != nil && !data.expErr) {
 			log.Fatalf("%s\ngot: '%v', want: '%v'", errMsg, err, data.expErr)
 		}
-		if !reflect.DeepEqual(c.MetaData, data.expMetadata) {
-			log.Fatalf("%s\ngot: '%v', want: '%v'", errMsg, c.MetaData, data.expMetadata)
+		if !cmp.Equal(c.metadata, data.expMetadata) {
+			log.Fatalf("%s\ngot: '%v', want: '%v'", errMsg, c.metadata, data.expMetadata)
```

Comment:
Not directly related to your changes, but while you're at it, how about printing the result of `cmd.Diff` instead of dumping those maps in the logs?